src/main/java/org/mongojack/internal/MongoJackInstantDeserializer.java

```diff
@@ -0,0 +1,42 @@
+package org.mongojack.internal;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.time.format.DateTimeFormatter;
+import java.util.Date;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonTokenId;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.datatype.jsr310.deser.InstantDeserializer;
+
+/**
+ * Patched Java8 Instant deserializer. Works with bson4jackson-deserialized ISODate() fields
```

Comment:
Can you change `Java8 Instant` to `java.time.Instant`? The current wording isn't quite clear whether Java8 refers to the Instant of the deserializer. I believe it's the Instant, but think that java.time.Instant would be a clearer what to say that